ff/serving/provider/offline_test.go

```diff
@@ -165,9 +214,9 @@ func testMaterializations(t *testing.T, store OfflineStore) {
 			SegmentStart: 0,
 			SegmentEnd:   3,
 			ExpectedSegment: []ResourceRecord{
-				{Entity: "a", Value: 1},
-				{Entity: "b", Value: 2},
-				{Entity: "c", Value: 3},
+				{Entity: "a", Value: 1, TS: time.UnixMilli(0).UTC()},
```

Comment:
Can you remove all these from the tests

Reply:
Right now have to leave them in for the "Expected" results. When the resource is written, its set to start of epoch time if TS wasnt initialized. So when the test result is returned, uninitialized resource TS will be queried and contain the start of epoch time, which requires TS: time.UnixMilli(0).UTC() to match. Only way to avoid this is to not scan TS, which we can't do, or remove timestamps that equal start of epoch which seems like odd behavior.